src/screens/GovernanceCall/GovernanceCallBanner.js

```diff
@@ -0,0 +1,222 @@
+      .then((result) => {
+        const thisDateNow = Date.now();
+        const prismicResponse = result?.results[0]?.data;
+        setTitle(prismicResponse?.title[0]?.text);
+        setDescription(prismicResponse?.description[0]?.text);
+        setGovernanceCallLink(prismicResponse?.link?.url);
+        const governanceCallStartTime = new Date(parseISO(prismicResponse?.scheduled_for));
+        const governanceCallEndTime = addHours(governanceCallStartTime, 1);
+        const timeTillEventStart = calculateLeftOverTime(governanceCallStartTime);
+        setRemainingTime(timeTillEventStart);
+        const isEventLive =
+          !isBefore(governanceCallEndTime, thisDateNow) && timeTillEventStart.minutesTillEventStart <= 0;
+
+        const formattedDate = formatDate(governanceCallStartTime, DATE_FORMAT);
+
+        const formattedTimeWithTimezone = formatDate(governanceCallStartTime, TIME_WITH_TIMEZONE_FORMAT);
+
+        const calculatedBannerVisibility =
+          (prismicResponse?.show_permanently && isAfter(governanceCallEndTime, thisDateNow)) ||
+          (timeTillEventStart.hoursTillEventStart <= 24 && timeTillEventStart.hoursTillEventStart >= 0);
+
+        setTimingData({
+          governanceCallStartTime,
+          governanceCallEndTime,
+          formattedDate,
+          formattedTimeWithTimezone,
+          isEventLive,
+        });
+
+        setIsVisible(calculatedBannerVisibility);
+
+        return true;
+      })
+      .catch((error) => reportErrorLog('Prismic content fetch failed', { error }));
+  }, []);
+
+  React.useEffect(() => {
+    fetchGovernanceCallDataMemoized();
+  }, [fetchGovernanceCallDataMemoized]);
+
+  React.useEffect(() => {
+    setInterval(() => {
+      if (timingData?.governanceCallStartTime !== undefined) {
+        setRemainingTime(calculateLeftOverTime(timingData?.governanceCallStartTime));
+      }
+    }, 60000);
+  }, [timingData]);
+
+  if (!isVisible) return null;
+
+  const showRequestModal = () => Linking.openURL(governanceCallLink);
+
+  const calculateRemainingTime = () => {
+    if (remainingTime.daysTillEventStart > 0) {
+      const remainingTotalHours = remainingTime.hoursTillEventStart - (remainingTime.daysTillEventStart * 24);
+      return `${remainingTime.daysTillEventStart}d ${remainingTotalHours}h`;
+    } else if (remainingTime.hoursTillEventStart > 0) {
+      return `${remainingTime.hoursTillEventStart}h`;
+    }
+    return `${remainingTime.minutesTillEventStart}m`;
+  };
+
+  return (
+    <TouchableContainer onPress={showRequestModal}>
+      <IconImage source={smartWalletIcon} />
+
+      <Column>
+        <Title numberOfLines={1}>{title}</Title>
+        {!!description && <Description>{description}</Description>}
+        {!timingData.isEventLive && (
+          <ScheduledCallTime>
+            {`${timingData.formattedDate} at ${timingData.formattedTimeWithTimezone}`}
+          </ScheduledCallTime>
+        )}
+      </Column>
+      {timingData.isEventLive && (
```

Comment:
Otherwise crash may occur
```suggestion
      {!!timingData.isEventLive && (
```

Reply:
Done